examples/pytorch/nlp/huggingface_models/question-answering/pruning/pytorch_pruner/eager/README.md

```diff
@@ -47,19 +47,22 @@ pruning:
 Please be awared that when the keywords appear in both global and local settings, we select the **local** settings as priority.
 ### Coding template:
 With a settled config file, we provide a template for implementing pytorch_pruner API:
+
 ```python
 model = Model()
 criterion = Criterion()
 optimizer = Optimizer()
 args = Args()
 
-from neural_compressor.experimental.pytorch_pruner.pruning import Pruning
+from neural_compressor.pruning.pytorch_pruner.pruning import Pruning
 
 pruner = Pruning("path/to/your/config.yaml")
 if args.do_prune:
-    pruner.update_items_for_all_pruners(start_step=int(args.sparsity_warm_epochs * num_iterations), end_step=int(total_iterations))  ##iterative
+    pruner.update_items_for_all_pruners(start_step=int(args.sparsity_warm_epochs * num_iterations),
+                                        end_step=int(total_iterations))  ##iterative
 else:
-   pruner.update_items_for_all_pruners(start_step=total_iterations+1, end_step=total_iterations+1) ## remove the pruner
+    pruner.update_items_for_all_pruners(start_step=total_iterations + 1,
```

Comment:
shall this func get exposed to external user? is it possible to hide this part?

Reply:
Some configs like start_step and end_step are affected by the training hyperparameters, which users should usually tune.  This function eases users to change these configs.  I suggest keeping it, but of course, removing it does not affect the functionality